pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -186,7 +186,7 @@ func (m *managerImpl) Start() error {
 				<-stop
 			}
 
-			time.Sleep(dbusReconnectPeriod)
+			<-time.After(dbusReconnectPeriod)
```

Comment:
If we don't use select, what is the difference between `<-time.After` and `time.Sleep`? Do we really need this simple change?